ChaosMod/Effects/db/Player/PlayerMonsterHead.cpp

```diff
@@ -0,0 +1,27 @@
+/*
+	Effect by Moxi
+*/
+
+#include <stdafx.h>
+
+static const Hash monsterHash = GET_HASH_KEY("sum_prop_ac_monstermask_01a");
+
+static void OnStart()
+{
+	LoadModel(monsterHash);
+
+	Ped player = PLAYER_PED_ID();
+	int bone = GET_PED_BONE_INDEX(player, 0x60F2);
+	Vector3 playerPos = GET_ENTITY_COORDS(player, false);
+
+	Object monsterHead = CREATE_OBJECT(monsterHash, playerPos.x, playerPos.y + 2, playerPos.z, true, false, false);
```

Comment:
Create it using "CreatePoolProp" instead -> no "SET_ENTITY_AS_NO_LONGER_NEEDED" needed then